classic-editor.php

```diff
@@ -331,12 +334,11 @@ public static function notice_after_upgrade() {
 			return;
 		}
 
-		$message = __( 'The Classic Editor plugin prevents use of the new Block Editor.', 'classic-editor' );
-
+		$message = __( 'Classic Editor plugin inhibits use of the new Block Editor if it is not properly configured.', 'classic-editor' ); # TWEAK: 1 - changed the "catastrophic" term "prevents" with the more "friendly-confidential" term "inhibit" 2 - more understanding anc concise description 3 - separation in a new line generic part of the warning with the specific setting action 4 - make the warning compatible with small screen size
 		if ( $settings['allow-users'] && current_user_can( 'edit_posts' ) ) {
-			$message .= ' ' . sprintf( __( 'Change the %1$sClassic Editor settings%2$s on your User Profile page.', 'classic-editor' ), '<a href="profile.php#classic-editor-options">', '</a>' );
+			$message .= ' ' . sprintf( __( '<br />Check your settings on %1$sPersonal Options%2$s profile page.', 'classic-editor' ), '<a href="profile.php#classic-editor-options">', '</a>' );

```

Comment:
Think the existing text is clearer. The place to change the settings is Users -> Profile. There is no "Personal Options" page. Also, no `<br>` is better, no need of two lines?

Reply:
> no need of two lines?

I think it's better to separate them aesthetically, because the text is long, and uses two lines anyway...

Reply:
<br> it can also be put externally to the text string...